### PR TITLE
Updated Carthage instructrions

### PR DESCRIPTION
**Pull request by unipheas**
Added instructions for importing the built Carthage frameworks into an Xcode project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,9 +19,44 @@ github "Dronecode/DronecodeSDK-Swift" ~> 0.3.0
 And then get the framework using:
 
 ```shell
+rm -rf Carthage ~/Library/Caches/org.carthage.CarthageKit ~/Library/Caches/carthage
```
Comment (julianoes) [inline README.md]: I would note that this cleaning the caches is sometimes needed if something goes wrong. I would probably not advise to always delete it no matter what.

Comment (JonasVautherin) [reply README.md]: Or maybe add that to a "Troubleshooting" section for Carthage?

Comment (unipheas) [reply README.md]: I like that.

**Comment by JonasVautherin** [comment]
Thanks!

**Comment by unipheas** [comment]
No problem!

**Pull request merged**